UIDokan/Sources/UIDokan/DesignSystem/DesignSystem.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  File.swift
+//  
+//
+//  Created by ziad on 05/07/2022.
+//
+
+import Foundation
+import UIKit
+ public struct DesignSystem{
+
+
+     //MARK: - Colors
+     public enum Colors: String{
```

Comment:
I have two comments here.
- Do we currently support all these colors? If not, we need to all only the colors we are supporting. 
- If you need to use it, you will have to use `let color = DesignSystem.Colors.blueOcean`. Instead, you may use extension on UIColor. Example: 
```swift
extension UIColor { 
    class var blueOcean: UIColor { 
        return UIColor(named: "Blue Ocean")!
    }
}
```
If you agree on the previous point, you may move it to Extensions.Colors